src/main/java/picocli/CommandLine.java

```diff
@@ -11581,6 +11583,21 @@ private static String extractName(ResourceBundle rb) {
                     return (String) ResourceBundle.class.getDeclaredMethod("getBaseBundleName").invoke(rb);
                 } catch (Exception ignored) { return "?"; }
             }
+            public Messages parent() {
+            	CommandSpec parentSpec = this.spec.parent();
+            	if (parent==null || parent.spec!=parentSpec) {
```

Comment:
Can we change this to have spaces before and after `==` and `!=`:
```
if (parent == null || parent.spec != parentSpec) {
```

Reply:
Done.